src/authentication/views/registration-flow/r3-stamboek.tsx

```diff
@@ -181,125 +183,133 @@ const RegisterStamboek: FunctionComponent<RegisterStamboekProps> = ({ history, l
 
 	return (
 		<Container className="c-register-stamboek-view" mode="vertical">
-			<Container mode="horizontal" size="small">
-				<div className="c-content">
-					<p className="u-text-bold">Heb je een stamboeknummer?</p>
-				</div>
-				<FormGroup>
-					<RadioButtonGroup>
-						<RadioButton
-							name="stamboeknummer"
-							value="yes"
-							label="Ja, ik heb een stamboeknummer / lerarenkaart nummer"
-							checked={intendsToHaveStamboekNumber}
-							onChange={setIntendsToHaveStamboekNumber}
-						/>
-						{intendsToHaveStamboekNumber === true && (
-							<Spacer
-								className="m-stamboek-input-field-wrapper"
-								margin={['left-large', 'top-small', 'bottom-large']}
-							>
-								<TextInput
-									placeholder="00000000000-000000"
-									value={rawStamboekNumber}
-									onChange={setStamboekNumber}
+			<Container mode="horizontal" size="large">
+				<Grid>
+					<Column size="2-7">
+						<div className="c-content">
+							<p className="u-text-bold">Heb je een stamboeknummer?</p>
+						</div>
+						<FormGroup>
+							<RadioButtonGroup>
+								<RadioButton
+									name="stamboeknummer"
+									value="yes"
+									label="Ja, ik heb een stamboeknummer / lerarenkaart nummer"
+									checked={intendsToHaveStamboekNumber}
+									onChange={setIntendsToHaveStamboekNumber}
 								/>
-								<Tooltip position="bottom" contentClassName="m-stamboek-tooltip">
-									<TooltipTrigger>
-										<span>
-											<Icon className="a-info-icon" name="info" size="small" />
-										</span>
-									</TooltipTrigger>
-									<TooltipContent>
-										<Spacer margin={'small'}>
-											<Spacer margin="bottom-small">
-												<span>Je stamboek nummer staat op je lerarenkaart</span>
-											</Spacer>
-											<img
-												alt="Voorbeeld leeraren kaart"
-												className="a-stamboek-image"
-												src="/images/leerkrachten-kaart-voorbeeld-nummer.png"
-											/>
+								{intendsToHaveStamboekNumber === true && (
```

Comment:
`=== true` is unnecessary.

Reply:
ik had die hier bijgezet omdat het bij de false undefined kan zijn en dan blijft het uniform, maar ik zal em weg doen bij de true